src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -90,6 +93,20 @@
                 (fn [req]
                   (ringutils/json-response 200
                                            (metrics-utils/mbean-names))))
+
+              (comidi/POST "" []
+                (fn [req]
+                  (try
+                    (let [metrics (with-open [reader (-> req :body io/reader)]
+                                    (json/parse-stream reader true))]
+                      (if (seq? metrics)
+                        (ringutils/json-response
+                         200 (map metrics-utils/get-mbean metrics))
```

Comment:
hahaha that was the first code I wrote actually, I think all 3 cases might be useful, string/map/array, but I didn't know what y'all would think. I'll add the other cases back in.